debian/rules

```diff
@@ -26,7 +26,7 @@ build-stamp: configure-stamp
 	dh_testdir
 
 	# Add here commands to compile the package.
-	$(MAKE) RELEASE=1 \
+	$(MAKE) RELEASE=1 PREFIX=/usr \
```

Comment:
this file is already present in main HEAD.

This patch has to be rebased on top of HEAD.